src/engraving/libmscore/tremolo.cpp

```diff
@@ -360,9 +372,6 @@ void Tremolo::layoutTwoNotesTremolo(double x, double y, double h, double spatium
     //---------------------------------------------------
     //   Step 1: Calculate the position of the tremolo (x, y)
     //---------------------------------------------------
-
-    y += (h - bbox().height()) * .5;
```

Comment:
This removal causes a MSVC compiler warning C4100 reg. "unreferenced formal parameter"  `double h`.